src/olympia/reviewers/models.py

```diff
@@ -1164,6 +1185,7 @@ def check_should_be_delayed(cls, version):
             content_review = None
         return (
             not is_langpack and
+            version.channel == amo.RELEASE_CHANNEL_LISTED and
```

Comment:
for all of these `check_...`methods, if the check is expensive (db queries) and it always only applies to listed versions we should probably be doing that check first to avoid unnecessary work for unlisted versions.

Reply:
Yeah. I had that in mind but for the moment there aren't that many extra db queries (a lot of the stuff fetched is useful at some point or another), so I favored readability, even if that meant an extra db query in some cases.